drivers/hmis_csv_importer/app/jobs/hmis_csv_importer/cleanup/expire_base_job.rb

```diff
@@ -0,0 +1,105 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# Base class for loader and importer expiration jobs
+#   Subclasses should define these methods
+#   def log_id_field
+#   end
+#
+#   def log_model
+#   end
+#
+#   def models
+#   end
+#
+module HmisCsvImporter::Cleanup
+  class ExpireBaseJob < ApplicationJob
```

Comment:
Thanks for DRYing this up.  I think there's a solid argument for running it once against `HmisCsvTwentyTwentyTwo.importable_files_map` and `HmisCsvTwentyTwenty.importable_files_map` to enforce the same retention policy against those tables.  Perhaps in the `TaskQueue`.  Definitely doesn't need to hold up this PR.